### ISSUE TITLE
[Accessibility][People Picker] Narrator announcing the people picker content twice

### ISSUE THREAD

**Issue by pkabadeI** [OfficeDev/office-ui-fabric-react#9422]
Environment Information
Package version(s): (5.122.3)
Browser and OS versions: (Chrome: 72.0.3626.121 (Official Build) (64-bit), Microsoft Edge: 44.17763.1.0, Microsoft Windows 1809)
Please provide a reproduction of the bug in a codepen:
Actual behavior:
Screen reader (Narrator) is announcing the names in the People picker twice. PFA

Expected behavior:
Screen reader should announce the content only once

Priorities and help requested:
Are you willing to submit a PR to fix? ( No)

Requested priority: (High)

Products/sites affected: (if applicable) 
![image](https://user-images.githubusercontent.com/16132588/59365179-bdf1fc80-8d55-11e9-90db-16e657094d15.png)

**Comment by natalieethell** [comment]
Hey @pkabadeI thanks for opening this issue! I'm not able to reproduce your issue in the PeoplePicker example on the [Fabric site (v.5)](https://developer.microsoft.com/en-us/fabric#/components/peoplepicker). After glancing at your screenshot of the DOM, it looks like you have two extra aria labels I'm not seeing on our example. Are those there by default for you? 

If you could provide a CodePen repro, that would be super helpful for debugging!

**Comment by natalieethell** [comment]
Here's a CodePen to get you started! https://codepen.io/slimemold/pen/mZerwW (thanks @joschect!)
